config/development/telemetry.kyma-project.io_metricpipelines.yaml

```diff
@@ -49,6 +49,13 @@ spec:
                   application:
                     description: Configures application related scraping.
                     properties:
+                      istio:
+                        description: Configures istio scraping.
```

Comment:
I think the description can be extended much more. Be short but precise on what will happen when the feature is in use.